packages/bsky/src/auth-verifier.ts

```diff
@@ -103,6 +115,17 @@ export class AuthVerifier {
           credentials: { type: 'standard', iss, aud },
         }
       } else if (isBearerToken(ctx.req)) {
+        // @NOTE temporarily accept entryway session tokens to shed load from PDS instances
+        const token = bearerTokenFromReq(ctx.req)
+        const header = token ? jose.decodeProtectedHeader(token) : undefined
+        if (header?.typ === 'at+jwt') {
```

Comment:
Love it that the `typ` header is coming in handy :-)

Reply:
Expilct all